In [1]:
name = "Svetlana"
hourly_wage = 25.50
hours = 30
weekly_pay = hourly_wage * hours
print(f"{name} earns {weekly_pay} per week")

Svetlana earns 765.0 per week


In [2]:
temps = [61, 65, 72, 58, 70, 74, 68]

warm_days = 0

for temp in temps:
    if temp > 65:
        warm_days = warm_days + 1

print(warm_days)

4


In [3]:
def to_celsius(f):
    return (f - 32) * 5/9

print(round(to_celsius(temps[0]), 1))

16.1


In [4]:
movie = {
    "title": "L.A. Confidential",
    "year": 1997,
    "rating": 9.0
}

for key, value in movie.items():
    print(key, ":", value)

title : L.A. Confidential
year : 1997
rating : 9.0


In [5]:
import pandas as pd
df = pd.read_csv("chickwts.csv", index_col=0)
df.head()

,weight,feed
1,179,horsebean
2,160,horsebean
3,136,horsebean
4,227,horsebean
5,217,horsebean


In [6]:
df.shape

(71, 2)

In [7]:
df.describe()

,weight
count,71.000000
mean,261.309859
std,78.073700
min,108.000000
25%,204.500000
50%,258.000000
75%,323.500000
max,423.000000


Twenty-five percent of the chicks weigh more than about 324 g, making them heavier than their remaining counterparts. The average weight of the chicks is about 261 g, and the median is 258 g. The chick weights vary significantly, with a standard deviation of about 78 g, and the middle 50% of the weights are between about 205 g and 324 g.